Apps/Scigraph/dwim/package.lisp

```diff
@@ -27,23 +27,235 @@ advised of the possiblity of such damages.
 
 (in-package :common-lisp-user)
 
-#+mcclim
-(eval-when (compile load eval)
+(eval-when (:compile-toplevel :load-toplevel :execute)
   (pushnew :clim-2 *features*))
 
```

Comment:
We certainly _did_, all over the place. Looks like there are still 3 spots where we use it. If we fix those, then this can go away.